documentation/release_notes_7.markdown

```diff
@@ -15,6 +15,34 @@ canonical: "/puppetdb/latest/release_notes.html"
 
 # PuppetDB: Release notes
 
+## PuppetDB 7.11.0
+
+Released July 26 2022
```

Comment:
Looks like August 2 is the new date? according to the #proj-puppet-releases channel